docs/src/test/java/docs/http/javadsl/server/directives/HeaderDirectivesExamplesTest.java

```diff
@@ -22,6 +22,9 @@
 import akka.http.javadsl.testkit.TestRoute;
 import scala.PartialFunction;
 
+import static akka.http.javadsl.server.Directives.*;
+import static akka.http.javadsl.common.PartialApplication.*;
```

Comment:
Yes, I need them to write something like
```scala
anyOf(
      bindParameter(this::headerValue, extractExampleHeader),
      bindParameter(this::provide, "newValue"),
     (String value) -> complete("header is " + value));
```

The alternative is more verbose coding myself the half applied functions.
Alternatively, I could just import the classes and reference the static methods explicitly.